src/users/users.graphql

```diff
@@ -20,6 +21,15 @@ input CreateUserInput {
   role: String
 }
 
+input EditUserInput {
+  firstName: String
```

Comment:
its should be nullable there and in validation so you don't need to provide all the fields, but on create you requres all fields

Reply:
I set all fields as nullable in entity to create admin only with login and password, but when a new user must be created there all fields should be filled